src/modules/string/index.ts

```diff
@@ -317,15 +317,17 @@ export class StringModule {
    *
    * @param options Either the number of characters or the options to use.
    * @param options.length The number or range of digits to generate. Defaults to `1`.
-   * @param options.allowLeadingZeros If true, leading zeros will be allowed. Defaults to `false`.
+   * @param options.allowLeadingZeros Whether leading zeros are allowed or not. Defaults to `true`.
    * @param options.exclude An array of digits which should be excluded in the generated string. Defaults to `[]`.
    *
+   * @see faker.number.int() if you would like to generate a number (within a range)
```

Comment:
```suggestion
   * @see faker.number.int() If you would like to generate a `number` (within a range).
```

Would make `within a value range` make it more distinct from the `length range`?